src/components/Portfolio/LoanFilterBar.vue

```diff
@@ -0,0 +1,82 @@
+<template>
+	<div>
+		<div class="tw-flex tw-flex-col">
+			<div class="tw-flex tw-justify-between tw-mb-2">
+				<div class="tw-text-sm tw-text-h2">
+					Filter loans
+				</div>
+				<kv-button
+					class="tw-text-sm"
+					variant="primary"
+				>
+					Export {{ totalLoans }} loans
+				</kv-button>
+			</div>
+			<kv-text-input
```

Comment:
It looks like we have a search box but it's not connected, are we adding search functionality?

Reply:
Yes, not sure if you saw my comment about the purpose of this PR: https://github.com/kiva/ui/pull/5781#issuecomment-2655132250

The search functionality backend doesn't exist yet.

Reply:
Sounds good, I wasn't sure how much the old comments still applied, thanks.